.github/workflows/build-perf-image-perf-test.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Create and publish a Docker image for performance tests
+on:
+  pull_request:
+    paths:
+      - 'performance/**'
+  push:
+    branches:
+      - 'main'
```

Comment:
Should this have a paths filter as well?  We probably don't need to rebuild on every push to main. :)

Reply:
Yeah, it really should.